pyvrp/search/tests/test_MoveTwoClientsReversed.py

```diff
@@ -67,7 +67,7 @@ def test_RC208_instance(rc208, seed: int):
     ls = LocalSearch(rc208, rng, compute_neighbours(rc208, nb_params))
     ls.add_node_operator(MoveTwoClientsReversed(rc208))
 
-    single_route = list(range(1, rc208.num_clients + 1))
+    single_route = list(range(rc208.num_depots, rc208.dimension))
```

Comment:
This comes up repeatedly and without the right knowledge it is not directly clear that `range(num_depots, dimension)` enumerates the client indices I think. Would it make sense to have a helper function? Or maybe (probably not let's at least consider the option) add `index` to `Client` such that you can iterate over `data.clients()`?

Reply:
>Would it make sense to have a helper function?

I want to do less of these, not more.

> Or maybe (probably not let's at least consider the option) add `index` to `Client` such that you can iterate over `data.clients()`?

Possibly, I've thought about this. But I don't really know how to do that without breaking things I like, like `Client` being immutable after construction.

Reply:
I think we can solve some of this with #384 as well.